__init__.py

```diff
@@ -8,4 +8,6 @@
 from LOGOS.src import CapitalInvestmentModel
 from LOGOS.src import BatteryReplacementCashFlowModel
 from LOGOS.src import IncrementalNPV
-from LOGOS.src.knapsack import BaseKnapsackModel
\ No newline at end of file
+from LOGOS.src.knapsack import MultipleKnapsackModel
+from LOGOS.src.knapsack import BaseKnapsackModel
```

Comment:
You do not need to import BaseKnapsackModel since this model can not be directly used by RAVEN.

Reply:
good point fixed